### ISSUE TITLE
[Bug] Scanning unidentified still scans identified games

### ISSUE THREAD

**Issue by Casuallynoted** [rommapp/romm#1000]
**RomM version**
3.3.0-rc.2

**Describe the bug**
At the moment, when scanning unidentified games, it still scans through games that have been identified.

**Comment by gantoine** [comment]
Wooho congrats on issue #1000! 🎉

**Comment by gantoine** [comment]
I can't seem to reproduce this with either a platform where all games are identified, or one where only some games are identified. :/

**Comment by Casuallynoted** [comment]
> I can't seem to reproduce this with either a platform where all games are identified, or one where only some games are identified...

Oh no! I've noticed this happening with my sufami turbo library, which is designated as a platform version for the super famicom. Maybe this is an issue that's exclusive to platform versions?

**Comment by gantoine** [comment]
Nope, tried it with an identified sufami turbo game, and selecting "Unidentified games" skips any game with igdb_id or moby_id set. I'm gonna have to close this as non-reproducible; looking over the code I can't see any way it would scan identified games with that option selected.
